docs/sources/writing-guide/front-matter/index.md

```diff
@@ -211,3 +211,31 @@ Aliases can be relative or absolute paths.
 Do not include an `aliases` entry that refers to the initial published website directory.
 The version in the URL path can cause undesirable redirects, such as a redirect from latest content to an old version.
 Aliases should be relative and not absolute paths so that old versions do not steal redirects from "latest" content when it is moved around.
+
+### Test an alias
+
+To test an alias results in the correct redirect, use your browser or a command-line tool for making HTTP requests.
+
+#### Using the browser
+
+1. Start the documentation webserver with `make docs`.
+1. Browse to the URL of the page that should be redirected.
+1. Confirm that you are redirected to the desired page.
```

Comment:
Suggest adding an example
```suggestion
1. Confirm that you are redirected to the desired page.

For example, if you want the page `https://grafana.com/docs/grafana/latest/panels/working-with-panels/` to redirect to `https://grafana.com/docs/grafana/latest/panels-visualizations/panel-editor-overview/`, navigate to the URL in the browser to confirm the redirect is working: 

```
http://localhost:3002/docs/grafana/latest/panels/working-with-panels/ 
```
```

Reply:
Thank you, added in 2d8b164